universal-login-commons/src/core/constants/test.ts

```diff
@@ -142,3 +142,7 @@ export const TEST_GAS_PRICES = {
     timeEstimation: '30',
   },
 } as unknown as GasPriceSuggestion;
+
+export const TEST_ENCRYPTED_WALLET_JSON = '{"address":"76010666821749b24145d6d792a3038a6de0cd71","id":"dc134bc2-9d47-4e73-990f-c9abeee94c41","version":3,"Crypto":{"cipher":"aes-128-ctr","cipherparams":{"iv":"7c4e7269f68a4b93337fd6f17d956735"},"ciphertext":"6897e05bf1088a1e83b5c8ca3648b55cbc417661de660d2a30da65223b5f2fd0","kdf":"scrypt","kdfparams":{"salt":"a8e44cd1357b86dcf387c29dd09f4f3b3d86ac6d411f3f88a069ef5f830b3c2e","n":131072,"dklen":32,"p":1,"r":8},"mac":"cc555dbde8816b36fbb4368fe3e9045c5a516a9028c51ada41860258b4da8ede"},"x-ethers":{"client":"ethers.js","gethFilename":"UTC--2020-07-09T14-31-30.0Z--76010666821749b24145d6d792a3038a6de0cd71","mnemonicCounter":"0d2f10e922bc048cdef52a543abbad3c","mnemonicCiphertext":"5eb6e83c13d073136f3125bbe47ee553","path":"m/44\'/60\'/0\'/0/0","version":"0.1"}}';
```

Comment:
We should make it typical JSON, not string.

Reply:
Done